src/smart-app-banner/smart-app-banner.tsx

```diff
@@ -0,0 +1,81 @@
+import {
+    ClickableContainer,
+    CloseContainer,
+    CrossButton,
+    CrossIcon,
+    Description,
+    DownloadButton,
+    DownloadContainer,
+    LifeSgAppIcon,
+    SmartAppBannerContainer,
+    TextContainer,
+    Title,
+} from "./smart-app-banner.styles";
+
+interface SmartAppBannerProps {
+    show: boolean;
+    link: string;
+    content: Content;
+    offset?: number;
```

Comment:
for optional props, add `undefined` type as well

Reply:
okay, added